service/src/main/java/uk/gov/hmcts/reform/fpl/logging/HeaderInformationExtractor.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.fpl.logging;
+
+import org.springframework.core.MethodParameter;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.PostMapping;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.joining;
+
+@Component
+public class HeaderInformationExtractor {
+
+    public String getCallback(CallbackRequest callbackRequest, MethodParameter parameter) {
+        String eventName = callbackRequest.getEventId();
+        String callbackType = Optional.ofNullable(parameter.getMethod())
+            .map(method -> method.getAnnotation(PostMapping.class))
+            .map(PostMapping::value)
```

Comment:
vv minor but is there a way that we can inject any path variables around here

Reply:
probably not without injecting the http request actually

Reply:
that's not used by the exception logger. Just moved in its own class